apps/dialtone-documentation/docs/components/empty-states.md

```diff
@@ -0,0 +1,378 @@
+status: ready
+thumb: true
+image: assets/images/components/empty-state.png
+figma: planned
+storybook: https://dialtone.dialpad.com/vue/?path=/story/components-card--default
+---
+
+<code-well-header>
+  <dt-empty-state
+    illustration-name="mind"
+    header-text="Nothing to see here"
+    body-text="Looks like there's no data to display here."
+  >
+    <template #body>
+      <div class="d-mt8 d-stack d-stack--row d-stack--gap-300">
+        <button class="d-btn">Click me</button>
+        <button class="d-btn d-btn--primary">Click me</button>
+      </div>
+    </template>
+  </dt-empty-state>
+</code-well-header>
+
+## Morphology & Anatomy
+
+Image with bullets
+<code-well-header>
+<img class="d-w90p" alt="empty state bullets" src="/assets/images/components/empty-state01.png">
+</code-well-header>
+
+1. Illustrations*
+2. Title
+3. Description*
+4. Actions
+
+- Primary (this primary action, can be a primary button or a muted one)
+- Secondary*
+
+5.Background
+
+*optionals
+
+The empty state uses all the space of the container it's replacing, always keeping the content in the middle.
+
+The space taken by the empty state can vary, the padding will be minimum 24px and can be flexible and adapt the container size.
+
+## Uses & Best practices
+
+We identify 4 types of empty states:
+
+<b> Zero state: </b> nothing has been added yet, we recommend to use this space to teach the user how to fill this container, or give a useful tip like a shortcut for something relatable or tell them what needs to happen if no action can be taken.
+
+<img class="d-w90p" alt="empty state example" src="/assets/images/components/empty-state02.png">
+
+<b> No results:</b> the user performed an action and there are no results, in this scenario the best usage of the empty state will be to recommend how to recover. If the user is filtering a table, recommend another filter that has results or, like the zero state case, how to create a new entry with the properties the user is filtering.
+
+<img class="d-w90p" alt="empty state example" src="/assets/images/components/empty-state03.png">
+
+<b> New feature:</b> A new section or screen the user doesn’t know and we want to introduce to them, if this action would include a change in their plan we should inform it in the component, also try to indulge/seduce the user to use this feature or to look interesting to them.
+
+<img class="d-w90p" alt="empty state example" src="/assets/images/components/empty-state04.png">
+
+<b> Not enabled:</b> the section isn’t available to the user at the moment.
+
+<img class="d-w90p" alt="empty state example" src="/assets/images/components/empty-state05.png">
+
+<dialtone-usage>
+<template #do>
+
+- Give the user all the information necessary to know how to proceed, but not as much for them to feel overwhelmed.
```

Comment:
"but not as much for them to feel overwhelmed." -> "but not so much they feel overwhelmed"